ironic/drivers/modules/drac/raid.py

```diff
@@ -884,6 +972,65 @@ def _check_node_raid_jobs(self, task):
                 self._clear_raid_config_job_failure(node)
                 self._set_clean_failed(task, config_job)
 
+        if finished_job_ids and not node.driver_internal_info.get(
+                'raid_config_job_failure'):
+            if 'raid_config_substep' in node.driver_internal_info:
+                if node.driver_internal_info['raid_config_substep'] == \
+                        'delete_reset_config':
+                    controllers = list()
+                    no_jobs_required = False
```

Comment:
I think it's confusing to have a variable name begin with "no".  Can you change the above variable name to "jobs_required" and set it to True?

Reply:
done